perun-cli/createAttribute

```diff
@@ -14,14 +14,12 @@ sub help {
 	--attributeName       | -N attribute friendly name
 	--attributeNameSpace  | -n namespace (e.g.: urn:perun:user:attribute-def:def)
 	--attributeDisplayName| -e attribute display name
-	--attributeType       | -t attribute type (integer/string/array/hash/boolean/largestring/largearray)*
+	--attributeType       | -t attribute type (integer/string/array/hash/boolean)*
```

Comment:
Since there is no comment, you should remove also ending ``*``.

Reply:
Fixed